tests/unit/services/test_sentry.py

```diff
@@ -48,6 +48,10 @@ def test_parse_without_envvar(event):
 
     data1 = event("sentry_event.json")
     data2 = event("sentry_event2.json")
+    data3 = event("sentry_event3.json")
```

Comment:
Would you mind renaming the two json files you've added to something that's more expressive, please? The existing names aren't very clear; I've no idea what sentry_event2.json is for and I'm sure I'll ask the same about the ones you've added in a few months too!

Reply:
Fair comment - I've got no idea about 1 or 2 so I'll leave them alone